package.json

```diff
@@ -4,7 +4,8 @@
   "description": "This is a workshop about Cypress",
   "main": "index.js",
   "scripts": {
-    "test": "cypress open"
+    "test": "cypress open",
```

Comment:
Leave only test:open script

Reply:
the guide has been updated, now is not required to delete the test script, since it will be modified in next sections